lisa/sut_orchestrator/azure/common.py

```diff
@@ -1226,70 +1223,44 @@ def get_storage_credential(
     return {"account_name": account_name, "account_key": key.value}
 
 
-def generate_blob_sas_token(
-    credential: Any,
-    subscription_id: str,
-    cloud: Cloud,
-    account_name: str,
-    resource_group_name: str,
+def generate_user_delegation_sas_token(
```

Comment:
It shouldn't use sas token anymore. The file should be able to access without sas, if the user has permission.

Reply:
I didn't find a way to copy vhd from one storage account into another storage account without sas in the same subscription, issue raised here https://github.com/Azure/azure-sdk-for-python/issues/35571 
Maybe we can convert the vhd into shared gallery image, then make replication when find a qualified location.